cmd/osbuild-image-tests/main_test.go

```diff
@@ -96,22 +96,22 @@ func splitExtension(path string) (string, string) {
 func testImageInfo(t *testing.T, imagePath string, rawImageInfoExpected []byte) {
 	var imageInfoExpected interface{}
 	err := json.Unmarshal(rawImageInfoExpected, &imageInfoExpected)
-	require.Nilf(t, err, "cannot decode expected image info: %#v", err)
+	require.NoErrorf(t, err, "cannot decode expected image info: %#v", err)
```

Comment:
For the uninitiated: what's the difference, why do you make this change?

Reply:
NoError() is more understandable (both in the code and in the output) than Nil() when asserting errors. Also the rest of the tests uses mostly NoError() when asserting errors, this change should lead to a better consistency.